src/main/java/de/teamlapen/vampirism/core/ModParticles.java

```diff
@@ -189,97 +186,33 @@ public Particle createParticle(World world, double posX, double posY, double pos
-                        float f1 = 1.0F - (float) (Math.random() * 0.30000001192092896D);
-                        this.particleRed = f1;
-                        this.particleGreen = f1;
-                        this.particleBlue = f1;
-                        this.particleScale *= 0.75F;
-                        this.particleScale *= 2.5F;
-                        this.oSize = this.particleScale;
-                        this.particleMaxAge = (int) (8.0D / (Math.random() * 0.8D + 0.3D));
-                        this.particleMaxAge = (int) ((float) this.particleMaxAge * 2.5F);
-                    }
-
-                    @Override
-                    public void renderParticle(BufferBuilder buffer, Entity entityIn, float partialTicks, float rotationX, float rotationZ, float rotationYZ, float rotationXY, float rotationXZ) {
-                        float f = ((float) this.particleAge + partialTicks) / (float) this.particleMaxAge * 32.0F;
-                        f = MathHelper.clamp(f, 0.0F, 1.0F);
-                        this.particleScale = this.oSize * f;
-                        super.renderParticle(buffer, entityIn, partialTicks, rotationX, rotationZ, rotationYZ, rotationXY, rotationXZ);
-                    }
-
-                    @Override
-                    public void onUpdate() {
-                        this.prevPosX = this.posX;
-                        this.prevPosY = this.posY;
-                        this.prevPosZ = this.posZ;
-
-                        if (this.particleAge++ >= this.particleMaxAge) {
-                            this.setExpired();
-                        }
-
-                        this.setParticleTextureIndex(7 - this.particleAge * 8 / this.particleMaxAge);
-                        this.move(this.motionX, this.motionY, this.motionZ);
-                        this.motionX *= 0.9599999785423279D;
-                        this.motionY *= 0.9599999785423279D;
-                        this.motionZ *= 0.9599999785423279D;
-                        EntityPlayer entityplayer = this.world.getClosestPlayer(this.posX, this.posY, this.posZ, 2.0D, false);
-
-                        if (entityplayer != null) {
-                            AxisAlignedBB axisalignedbb = entityplayer.getEntityBoundingBox();
-
-                            if (this.posY > axisalignedbb.minY) {
-                                this.posY += (axisalignedbb.minY - this.posY) * 0.2D;
-                                this.motionY += (entityplayer.motionY - this.motionY) * 0.2D;
-                                this.setPosition(this.posX, this.posY, this.posZ);
-                            }
-                        }
-
-                        if (this.onGround) {
-                            this.motionX *= 0.699999988079071D;
-                            this.motionZ *= 0.699999988079071D;
-                        }
-                    }
-                };
+                IParticleFactory par = new ParticleCloud.Factory();
```

Comment:
Is it possible to store the ParticleCloud factory as a  (final) variable inside this factory class/object?